src/crystal/system/unix/lib_event2.cr

```diff
@@ -7,6 +7,8 @@ require "c/netdb"
 {% if flag?(:openbsd) %}
   @[Link("event_core")]
   @[Link("event_extra")]
+{% elsif compare_versions(Crystal::VERSION, "0.35.0-0") >= 0 %}
```

Comment:
```suggestion
# TODO: remove after v0.35.0
{% elsif compare_versions(Crystal::VERSION, "0.35.0-0") >= 0 %}
```

Reply:
Is just one version back still our policy? Without this Crystal <= 0.34 cannot compile Crystal >= 0.35

Reply:
My main point was to add a `TODO` note, specifying the right version is another story, which you probably know better than I do.

Reply:
My question is, is it really safe to remove the check after 0.36 already?

Reply:
Yes @jhass it's fine. One version for the compiler/std-lib is what is needed.

@Sija since we always use `compare_versions(Crystal::VERSION` I see no reason to add a comment; it's easily searchable. Is a matter of taste of course. On deprecation warnings, it's another story because we can schedule the removal to something not stated in the annotation.